stock_prices/stock_prices.py

```diff
@@ -1,15 +1,45 @@
 #!/usr/bin/python
 
+# buy sell amazon stock
+# receives input list of stock prices
+# function returns max profit from a single buy and sell
+# must buy first before selling
+
+# have two loops, initialize min and max to 0, profit to 0
+# set min val = i
+# set max val = j
+# if max val - min val is greater than max, reassign new profit max
+
 import argparse
 
+
 def find_max_profit(prices):
-  pass
+    maxVal = 0
+    minVal = 0
+    profit = 0
+    for i in range(0, len(prices)-1):
+        minVal = prices[i]
+        print(f" index of min: {i}. value of min:{minVal}")
+        for j in range(i+1, len(prices)):
```

Comment:
this works great! however, your solution right now is O(n**2). After you're done with MVP, see if you can get it down to O(n)